<launch>

    <arg name="robot_name" default="sam"/>

    <group ns="$(arg robot_name)">
        <group ns="dr">

    	<node name="odom_listener" pkg="tf_convenience_topics" type="odom_listener" output="screen" >
	    	<param name="odom_topic" value="/$(arg robot_name)/dr/odom"/>
	    	<param name="topic_from_dvl" value="/$(arg robot_name)/core/dvl"/>
        	<param name="loop_freq" value="10" />
    	</node>

    	</group>
   </group>

</launch>
